h/static/scripts/tests/controllers/character-limit-controller-test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+var CharacterLimitController = require('../../controllers/character-limit-controller');
+var util = require('./util');
+
+describe('CharacterLimitController', function () {
+
+  /**
+   * Make a <textarea> with the character limit controller enhancement
+   * applied and return the various parts of the component.
+   */
+  function component(value, maxlength) {
```

Comment:
This is fine, though unfortunately verbose. Once my other open PRs land we'll be able to simplify this to something like:

```
function component(value, maxlength = 250)
  var template = `<textarea class="js-character-limit" data-maxlength="${maxlength}">${value}</textarea>
<span class="js-character-counter">Foo</span>`;

  return util.setupComponent(document, template, CharacterLimitController);
}

var ctrl = component("test", 42);
assert.equal(ctrl.refs.characterCounter.textContent, "4 / 42")
```